charts/prometheus-node-exporter/Chart.yaml

```diff
@@ -1,8 +1,9 @@
-apiVersion: v1
-appVersion: 1.1.2
+apiVersion: v2
+appVersion: 1.2.0
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.18.2
+version: 1.19.0
```

Comment:
Can move this to 2.0.0 as well if desired.

Reply:
Since we're breaking backwards-compatibility with Helm v2 it would be a good idea I think! 👍

Reply:
Will do!